tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -0,0 +1,23 @@
+@api
+Feature: search
```

Comment:
does this file search using tag only? If yes we could mention it in the file name, line 4 and 5

Reply:
No, this file search is not only using tag only but also using context